pyaedt/edb_core/EDB_Data.py

```diff
@@ -726,8 +726,7 @@ def __init__(self, edb_padstack, layer_name, pad_type, parent):
 
     @property
     def geometry_type(self):
-        """Retrieve the geometry type.
-    
+        """
```

Comment:
We have to have shot descriptions for these methods.  ``Geometry type.`` is sufficient.

Reply:
Just saw you added this back in.

Reply:
@akaszynski  So do this for all with the @property? I assume that I will continue to see syntax errors for subsequent entries if I don't. My notes say that @properties indicates that there are no parameters for this, but yet some of the @properties do have them?!

Reply:
@akaszynski I shorted to Geometry type--but it still failed at the same location. Other earlier @property items have short descriptions that are just as long as this one was with the "Retrieve" portion. I can't figure out what is wrong!

Reply:
Figured it out, just a missing `"` in the class docstring above this.  I've committed and pushed the change to get the CI rolling.

As for `@properties` with parameters, could you please point out where that occurs?  I'd like to take a look at that.